ores/scoring_systems/tests/util.py

```diff
@@ -49,6 +49,7 @@ def test_scoring_system(scoring_system):
         ScoreRequest("fakewiki", [1], ["fake"],
                      injection_caches={1: {wait_time: 0.05}},
                      model_info=['version']))
+    print(response.errors)
```

Comment:
This doesn't look right, it should have some notes or something like that.

Reply:
Thanks, I commented.